### PR TITLE
update tools for running local bmo

### PR DESCRIPTION
**Pull request by dhellmann**
Have the run script build a development version of the deployment for
metal3 by looking at the production one and removing the bmo and some
other settings we don't want.

Wait longer for the new pod to run, since we now wait for the init
container to download RHCOS.

### CODE REVIEW

**Comment by stbenjam** [comment]
/approve

I also just noticed I made a typo in the owners: https://github.com/openshift-metal3/dev-scripts/pull/1042

File: metal3-dev/run.sh [hunk 1]
```diff
@@ -1,17 +1,64 @@
...
+source $SCRIPTDIR/utils.sh
+
 if ! which operator-sdk 2>&1 >/dev/null ; then
-    echo "Did not find operator-sdk, set INSTALL_OPERATOR_SDK=1 in config_$USER.sh"
+    echo "Did not find operator-sdk." 1>&2
+    echo "Install it following the instructions from the baremetal-operator repository." 1>&2
+    exit 1
+fi
+
+if ! which yq 2>&1 >/dev/null ; then
```
Comment (hardys) [inline metal3-dev/run.sh:17]: Note that https://github.com/openshift-metal3/dev-scripts/issues/888 is still open, so I'm not sure anything will install yq on a freshly installed box?

I'd hoped we might remove the dependency by reworking the template mangling in `utils.sh` but if we're adding more usage then we should probably fix that - I guess we could just pip install it, or do pip install --user?

**Comment by hardys** [comment]
lgtm other than the concern that yq may not be installed on a freshly installed box - @dhellmann what are your thoughts, should we go ahead and land this then fix that as a follow-up?

File: metal3-dev/run.sh (hunk 1 above)
Comment (dhellmann) [reply metal3-dev/run.sh:17]: I can submit another PR to install it as part of the requirements script, if you think that's reasonable.

Comment (dhellmann) [reply metal3-dev/run.sh:17]: See #1044

**Pull request merged**